openedx/core/djangoapps/user_api/accounts/views.py

```diff
@@ -202,3 +205,23 @@ def partial_update(self, request, username):
             )
 
         return Response(account_settings)
+
+
+class AccountDeactivationView(APIView):
+    """
+    Account deactivation viewset. Currently only supports POST requests.
+    Only admins can deactivate accounts.
+    """
+    permission_classes = (permissions.IsAuthenticated, IsStaffUser)
```

Comment:
Staff users (e.g. nearly everyone in 141 Portland) should not be able to deactivate user accounts. This capability should be reserved for superusers and those with explicit access. I would use a [custom permission](https://docs.djangoproject.com/en/1.10/ref/models/options/#permissions), `can_deactivate_users` (Can deactivate, but NOT delete users), for this purpose.